web3/personal/api.go

```diff
@@ -0,0 +1,95 @@
+package personal
```

Comment:
would be nice a comment explaining what's `personal` package about :)

Reply:
added README.md